rlw/metadata.rb

```diff
@@ -2,7 +2,7 @@
 maintainer  "RightScale, Inc."
 license     "see LICENSE file in repository root"
 description "Base scripts for RightLink10 on Windows (RLL) to initialize basic functionality"
-version     '10.1.5'
+version     '10.1.4'
```

Comment:
So think this should be kept at 10.1.5 for the time being, so it doesn't conflict with the release